### DISCUSSION TITLE
changing the console background color

### DISCUSSION THREAD

**Discussion by kevison** [spectreconsole/spectre.console#1112] [Q&A] [answered]
I have my command prompt's background color set to a very dark green.  I am trying to do the same with AnsiConsole, but setting AnsiConsole.Background only changes the background of the words that are printed... the rest of the background of the console is always black... How can I make the default background of the console a different color?
thanks!

**Comment by patriksvensson** [comment, marked as answer]
That's not possible with Spectre.Console since there is no VT100 code for doing that.
If your terminal has an API, you will need to call that specifically.

However, Not using any background color when writing text should use your terminal's default background color.
